packages/blockchain-wallet-v4-frontend/src/services/CheckAccountZeroBalance/index.ts

```diff
@@ -0,0 +1,19 @@
+import { SwapAccountType, SwapCoinType } from '../../data/components/swap/types'
```

Comment:
I think you can just import `data/components/swap/types`

Reply:
Fixed!